lib/active_model_serializers/model.rb

```diff
@@ -0,0 +1,43 @@
+# ActiveModelSerializers::Model is a convenient
+# serializable class to inherit from when making
+# serializable non-activerecord objects.
+module ActiveModelSerializers
+  class Model
+    include ActiveModel::Model
+    include ActiveModel::Serialization
```

Comment:
See what happens if you take it out.  (Didn't I [suggest that in the first place?](https://github.com/rails-api/active_model_serializers/pull/1241#discussion_r41398549)) Yes, it's required.  What I'd rather have if instead to `include ActiveModel::Serializers::JSON` (which includes `ActiveModel::Serialization` but our serializers have strayed from that interface, unfortunately.